src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -21,43 +21,43 @@
     @FXML
     private Label id;
     @FXML
-    private Label phone;
+    private HBox startEndOnly;
```

Comment:
I'm not sure if the variable name ending with `only` is suitable. I think that can be changed in the future though.